iOSApp/Screens/Registration/RegistrationViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  RegistrationViewController.swift
+//  iOSApp
+//
+//  Created by Inas Ahmic on 17. 3. 2021..
+//
+
+import UIKit
+
+class RegistrationViewController: UIViewController, UITextFieldDelegate {
+
+    @IBOutlet weak var email: CustomTxtField!{
+        didSet{
+            email.delegate = self
+        }
+    }
+    
+    @IBOutlet weak var username: CustomTxtField!{
+        didSet{
+            username.delegate = self
+        }
+    }
+    
+    @IBOutlet weak var password: CustomTxtField!{
+        didSet{
+            password.delegate = self
+        }
+    }
+    
+    
+    @IBAction func next(_ sender: Any) {
+        let storyboard = UIStoryboard(name: "Registration", bundle: nil)
+        let viewcontoroller = storyboard.instantiateViewController(identifier: "RegistrationSecondViewController")
+        self.present(viewcontoroller, animated: true, completion: {
+            
+        })
+    }
+    
+    @IBAction func backToLogin(_ sender: Any) {
+        let storyboard = UIStoryboard(name: "Login", bundle: nil)
+        let viewcontoroller = storyboard.instantiateViewController(identifier: "LoginViewController")
+        self.present(viewcontoroller, animated: true, completion: {
+            
+        })
+        
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let tap = UITapGestureRecognizer(target: self, action: #selector(UIInputViewController.dismissKeyboard))
+        view.addGestureRecognizer(tap)
+
+        // Do any additional setup after loading the view.
+    }
+    
+    @objc func dismissKeyboard() {
+        
+        view.endEditing(true)
+    }
+    
+    @objc func tapBack (_ sender: UITapGestureRecognizer) {
+        let storyboard = UIStoryboard(name: "Login", bundle: nil)
+        let viewcontoroller = storyboard.instantiateViewController(identifier: "LoginViewController")
+        self.present(viewcontoroller, animated: true, completion: {
+            //print("screen is presented")
+        })
+    }
+    
+    func checkPassowrd(password string: String) {
+        if (string.count < 8) {
```

Comment:
password should contain the numbers as well as characters, please add this logic to your validation